### PR TITLE
TR2 Test Fix | LPS-174560 | master

### PR DESCRIPTION
**Pull request by luanakarla**
Ticket:
https://issues.liferay.com/browse/LPS-174560
https://issues.liferay.com/browse/LPS-159528

### CODE REVIEW

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:site-initializer-testray

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase [hunk 1]
```diff
@@ -16,212 +19,255 @@ definition {
...
 
 		if (${testPortalInstance} == "true") {
 			PortalInstances.tearDownCP();
 		}
 	}
 
-	@description = "This is a test for LPS-156998 - Verify if as an Administrator I should be able to add and delete a Component that doesn't have a Team"
+	@description = "Story ID: LPS-159528 | Test Summary: Verify if as an Administrator I should be able to add and delete a Component that doesn't have a Team"
 	@priority = 5
 	test AdminCanAddAndDeleteComponentWithoutTeam {
```
Comment (nikki-pru) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase:54]: Separar por favor.

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase [hunk 2]
```diff
@@ -16,212 +19,255 @@ definition {
...
+			Testray.deleteComponent(itemName = ${componentName});
+
+			Testray.assertItemNotPresent(itemName = ${componentName});
 		}
 	}
 
-	@description = "This is a test for LPS-156997 - Verify if as an Administrator I should be able to add and delete a Component that has a Team"
+	@description = "Story ID: LPS-159528 | Test Summary: Verify if as an Administrator I should be able to add and delete a Component that has a Team"
 	@priority = 5
 	test AdminCanAddAndDeleteComponentWithTeam {
```
Comment (nikki-pru) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase:86]: Separar pro favor. Um test, um intencao.

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase [hunk 3]
```diff
@@ -16,212 +19,255 @@ definition {
...
 		task ("Then the component should appear on the list") {
-			Testray.assertItemName(text = "Components_1 Project");
+			Testray.assertItemName(text = ${componentName});
 		}
 	}
 
-	@description = "This is a test for LPS-156994 - Verify if as an Administrator I am able to edit Components and change the assigned Team"
+	@description = "Story ID: LPS-159528 | Test Summary: Verify if as an Administrator I am able to edit Components and change the assigned Team"
 	@priority = 4
 	test AdminCanEditChangeComponent {
```
Comment (nikki-pru) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase:148]: Separar tambem por favor.

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase [hunk 4]
```diff
@@ -16,212 +19,255 @@ definition {
...
 		task ("Then the team should be updated on the list") {
-			Testray.assertItemName(text = "Components_1 Team Second");
+			Testray.assertItemName(text = "Team 2");
 		}
 	}
 
-	@description = "This is a test for LPS-156988 - Verify that as an Administrator I am able to edit Components while connecting a Team"
+	@description = "Story ID: LPS-159528 | Test Summary: Verify that as an Administrator I am able to edit Components while connecting a Team"
 	@priority = 5
 	test AdminCanEditComponent {
```
Comment (nikki-pru) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase:204]: AdminCanAddTeamToComponent

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase [hunk 5]
```diff
@@ -239,20 +285,26 @@ definition {
 
 	}
 
-	@description = "This is a test for LPS-162505 - Verify if as an Administrator I can see the warning message 'This field is required' when trying to save without a name"
+	@description = "Story ID: LPS-159528 | Test Summary: Verify if as an Administrator I can see the warning message 'This field is required' when trying to save without a name"
 	@priority = 4
 	test AdminCanViewWarningMessage {
```
Comment (nikki-pru) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase:290]: CannotAddComponentWithoutAName

**Review by nikki-pru — COMMENTED** [review]
Otimo pra trabalho adicionar Story ticket e GWT e Fluxo.

Pouca mudanca sobre Test Names. Lembrar intencao.

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase (hunk 5 above)
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase:290]: Done. Thanks!

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase (hunk 4 above)
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase:204]: Done. Thanks!

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase (hunk 2 above)
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase:86]: Done. Thanks.

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase (hunk 3 above)
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase:148]: Done. Thanks!

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase (hunk 1 above)
Comment (luanakarla) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayComponentsAdministrator.testcase:54]: Done. Thanks!

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:test:sf

**Comment by luanakarla** [comment]
ci:forward

**Pull request closed without merging**